src/utils/formatBalance.ts

```diff
@@ -5,8 +5,8 @@ export const getBalanceNumber = (balance: BigNumber, decimals = 18) => {
   return displayBalance.toNumber()
 }
 
-export const getFullDisplayBalance = (balance: BigNumber, decimals = 18) => {
-  return balance.dividedBy(new BigNumber(10).pow(decimals)).toFixed()
+export const getFullDisplayBalance = (balance: BigNumber, decimals = 18, decimalsToAppear?: number) => {
```

Comment:
Separated this into its own PR but needed the work in here during development
https://github.com/pancakeswap/pancake-frontend/pull/968

Reply:
#968 has been merged, you can update this now.

Reply:
Noice, cleared 👍